project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.9
+sbt.version=1.6.2
```

Comment:
#930 upgraded various docker base images to `hseeberger/scala-sbt:eclipse-temurin-11.0.14.1_1.6.2_2.13.8` - which uses sbt 1.6.2. However, this build.properties still specified 1.4.9. Let's fix that.